src/internal_events/pulsar.rs

```diff
@@ -33,3 +33,26 @@ impl InternalEvent for PulsarSendingError {
         });
     }
 }
+
+pub struct PulsarPropertyExtractionError<'a> {
+    pub property_field: &'a str,
+}
+
+impl InternalEvent for PulsarPropertyExtractionError<'_> {
+    fn emit(self) {
+        error!(
+            message = "Failed to extract properties. Value should be a map of String -> Bytes.",
+            error_code = "extracing_property",
+            error_type = error_type::PARSER_FAILED,
+            stage = error_stage::RECEIVING,
+            property_field = self.property_field,
+            internal_log_rate_secs = 10,
```

Comment:
```suggestion
            internal_log_rate_limit = true,
```

There was a change a month or so ago to make this a boolean and a global config.